internal/assemble/assemble.go

```diff
@@ -29,6 +29,36 @@ func assembleInput(ti typeNameToInfo, p *parse.InputPart) error {
 	return fmt.Errorf("unknown type: %s", p.Source.Prefix)
 }
 
+type sourceField int
+
+const (
+	fromString sourceField = iota
+	fromName
+)
+
+// createOutCols creates the list of output columns from the parameters passed
+// in.
```

Comment:
In the go style guide it has some stuff about how comments should be formatted. It suggests not having a strict line limit and trying to keep comments in semantic groups. https://google.github.io/styleguide/go/decisions#comment-line-length . If we follow this we should probably have the whole first sentence on line 39.

Reply:
Done.